cleverpush/src/main/java/com/cleverpush/NotificationOpenedProcessor.java

```diff
@@ -20,11 +22,16 @@ public static void processIntent(Context context, Intent intent) {
     Gson gson = new Gson();
     Notification notification = gson.fromJson(intent.getStringExtra("notification"), Notification.class);
     Subscription subscription = gson.fromJson(intent.getStringExtra("subscription"), Subscription.class);
+    String actionIndex = intent.getStringExtra("actionIndex");
+    int notification_Id = intent.getIntExtra("notificationId", 0);
```

Comment:
naming convention notification_Id -> notificationId i think?

Reply:
rename to requestId?

Reply:
I have changed notification_Id  -> activeNotificationId because we already declared notificationId in scope.